scalableminds/webknossos
# Collapse and Expand all groups feature

This PR adds two additional buttons to the small dropdown menu of all groups in the tree hierarchy view. These buttons make it possible to collapse and expand all groups.

### URL of deployed dev instance (used for testing):
- [https://addcollapseallgroups.webknossos.xyz/](https://addcollapseallgroups.webknossos.xyz/)

### Steps to test:
- Open a skeleton/hybrid tracing and create some groups and trees. Make sure you also put groups into each other to also test "complex" hierarchies.
- Use the two new buttons in the dropdown menu of any group. This should collapse or expand all subgroups. 

### Issues:
- fixes #4115

------
